app/controllers/v1/application_controller.rb

```diff
@@ -3,5 +3,11 @@
 # Superclass for all V1 controllers
 module V1
   class ApplicationController < ::ApplicationController
+    def rbac_allowed?
+      return true if ActiveModel::Type::Boolean.new.cast(Settings.disable_rbac)
+      return valid_cert_auth? if identity_header.cert_based?
+
+      user.authorized_to?(self.class.action_permission action_name.to_sym)
```

Comment:
Implicit check for `INVENTORY_READ`:

```suggestion
      user.authorized_to?(self.class.action_permission(action_name.to_sym), Rbac::INVENTORY_READ)
```

Reply:
Yeah, consuming `action_permission(s)` like this won't work anymore. You should probably use `self.class.instance_variable_get(:@action_permissions)`.